x/did/keeper/msg_server_did.go

```diff
@@ -65,8 +65,11 @@ func (m msgServer) DeactivateDID(goCtx context.Context, msg *types.MsgDeactivate
 		return nil, sdkerrors.Wrapf(types.ErrDIDDeactivated, "DID: %s", msg.DID)
 	}
 
-	signableDID := types.SignableDID(msg.DID)
-	newSeq, err := VerifyDIDOwnership(signableDID, docWithSeq.Seq, docWithSeq.Document, msg.VerificationMethodID, msg.Signature)
+	doc := types.DIDDocument{
```

Comment:
Instance of using the did, use the DIDDocument

Reply:
Because did is string. But not implements protobuf marshal

Reply:
Yeah. It makes sense even though the user must get a DID document before deactivating it.

Reply:
Yeah. It makes sense.